common-pvt.yml

```diff
@@ -225,17 +225,25 @@ buildvariants:
     tasks: *atlas_live_tasks
 
   #######################################
-  #           macOS Buildvariant        #
+  #         macOS Buildvariants         #
   #######################################
 
-  - name: macOS-1014-ssl
-    display_name: MacOS 10.14 SSL
+  - name: macOS-1014-x86_64-ssl
+    display_name: MacOS 10.14 x86_64 SSL
```

Comment:
(question) Why are we saying MacOS 10.14 here? The latest macOS versions can definitely run on x86_64, but I'm also not sure where we display this, or if we do special things that only work on versions prior to 10.15 (which seems unlikely to me).

Reply:
Yeah I think that's what happened. We set version compatibility here: https://github.com/mongodb/mongo-tools/blob/e62cf8866c5b98c45faf2fbe354703969aa076c6/set_goenv.sh#L24-L33

So nothing special for 10.14. We should upgrade if we can.

Reply:
macOS 10.15 works, but 11.00 consistently [breaks](https://spruce.mongodb.com/version/643dac32850e61bd621796dc/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC). I’ve left it at 10.15; I’ll open a separate investigation into why 11.00 fails.